XCode/EndlessSky-Info.plist

```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleSignature</key>
```

Comment:
Shouldn't this be changed to 0.9.17? Or, not because it won't let you add `alpha` or whatever?
Same for the manifest.

Reply:
Yeah I can't add `alpha`, so what we usually do is we just let the previous version number there.

Reply:
Merge everything!